README.md

```diff
@@ -26,15 +22,19 @@ To update the tests snapshots use
 $ npm t -- -u  # like this
 $ yarn test -u # or this
 ```
-
-## Configuration
-
-TODO
-
 ## Troubleshooting
 
 You can check if others are experiencing similar issues [here](https://github.com/vtex-apps/product-summary/issues). Also feel free to [open issues](https://github.com/vtex-apps/product-summary/issues/new).
 
-## Contributing
-
-TODO
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `product`          | `Product`  | Product that owns the informations                                          |
```

Comment:
You must do another table with the product props

Reply:
👍

Reply:
👍